database/pgx/v5/pgx_test.go

```diff
@@ -35,11 +35,11 @@ var (
 		PortRequired: true, ReadyFunc: isReady}
 	// Supported versions: https://www.postgresql.org/support/versioning/
 	specs = []dktesting.ContainerSpec{
-		{ImageName: "postgres:12", Options: opts},
```

Comment:
Confirming that postgres 12 will no longer have releases:
https://www.postgresql.org/support/versioning/